apps/rig/lib/rig/application.ex

```diff
@@ -2,10 +2,15 @@ defmodule Rig.Application do
   @moduledoc false
 
   use Application
+  use Rig.Config, []
 
   def start(_type, _args) do
     alias Supervisor.Spec
 
+    # Override application logging with environment variable
+    conf = config()
+    Logger.configure [{:level, conf.log_level}]
```

Comment:
- not sure if we need to use `conf = ..` even in cases where we need it only once... I think I'd be okay with `config().log_level`, but ymmv
- formatting: parens for `configure` function call